themes/10up-theme/theme.json

```diff
@@ -1,5 +1,6 @@
 {
-  "version": 1,
+  "$schema": "https://schemas.wp.org/wp/6.0/theme.json",
```

Comment:
We should use 6.1 at this point :)

Reply:
Thanks for catching this! I pushed a new commit that updates the schema to 6.1